config/schedule.rb

```diff
@@ -28,3 +27,9 @@
   honeybadger_wrapped_script 'index_sirsi_full.sh new', sirsi_server: 'morison', solr_url: '${MORISON_SOLR_URL}'
   honeybadger_wrapped_script 'index_sirsi_full.sh new', sirsi_server: 'bodoni', solr_url: '${SOLR_URL}'
 end
+
+
+every '*/15 1-6 * * *' do
+  honeybadger_wrapped_script 'commit.sh', solr_url: '${MORISON_SOLR_URL}'
+  honeybadger_wrapped_script 'commit.sh',solr_url: '${SOLR_URL}'
```

Comment:
Does this need a space before solr_url `honeybadger_wrapped_script 'commit.sh',solr_url: '${SOLR_URL}'` ?

Reply:
Thanks, fixed.